Entities/ACraft.cpp

```diff
@@ -806,6 +809,31 @@ bool ACraft::OnMOHit(MovableObject *pOtherMO)
 }
 
 
+/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+void ACraft::GibThis(Vector impactImpulse, float internalBlast, MovableObject* pIgnoreMO) {
+
+	if (g_SettingsMan.EnableCrabBombs() && !m_sCrabBombInEffect) {
+		m_sCrabBombInEffect = true;
+		unsigned short crabCount = 0;
```

Comment:
Turns out it's pretty nonsensical in modern computing to use unsigned shorts and stuff. Just use an int here